lib/whoop/formatters/sql_formatter.rb

```diff
@@ -6,6 +6,17 @@
 module Whoop
   module Formatters
     module SqlFormatter
+      # Hash of patterns to preserve in the SQL. The key is the expected pattern,
+      # the value is the pattern after it has been "broken" by anbt formatting.
+      # Instances of the value are replaced by the key.
+      PATTERNS_TO_PRESERVE = {
+        '::': ' : : ',
+        '->>': '- > >',
+        '->': '- >',
+        '#>>': '# > >',
+        '#>': '# >'
+      }
```

Comment:
This is great. I like the comment explaining why we are providing this fix.

I recommend a couple of things:

1. Add an additional explanation that the following patterns are for PostgreSQL jsonb operators and add a link to the formatters docs (on postgres website).
2. I would format the `PATTERNS_TO_PRESERVE` to use hash rockets for easier developer legibility.

Example:

```
PATTERNS_TO_PRESERVE = {
  "::"  => " : : ",
  "->>" => "- > >",
  "->"  => "- >",
  "#>>" => "# > >",
  "#>"  => "# >"
}
```